sdv/datasets/demo.py

```diff
@@ -197,21 +197,65 @@ def _extract_data(bytes_io, output_folder_name):
             return in_memory_directory
 
 
-def _get_data(modality, output_folder_name, in_memory_directory):
+def _get_data_with_output_folder(output_folder_name):
+    """Load CSV tables from an extracted folder on disk.
+
+    Returns a tuple of (data_dict, skipped_files).
+    Non-CSV files are ignored.
+    """
     data = {}
-    if output_folder_name:
-        for root, _dirs, files in os.walk(output_folder_name):
-            for filename in files:
-                if filename.endswith('.csv'):
-                    table_name = Path(filename).stem
-                    data_path = os.path.join(root, filename)
-                    data[table_name] = pd.read_csv(data_path)
+    skipped_files = []
+    for root, _dirs, files in os.walk(output_folder_name):
+        for filename in files:
+            if not filename.lower().endswith('.csv'):
+                skipped_files.append(filename)
+                continue
+
+            table_name = Path(filename).stem
+            data_path = os.path.join(root, filename)
+            try:
+                data[table_name] = pd.read_csv(data_path)
+            except UnicodeDecodeError:
+                data[table_name] = pd.read_csv(data_path, encoding='latin-1')
```

Comment:
why 'latin-1'?

Reply:
It maps every byte to a Unicode code point, ie the decoding never fails. It's also just a reasonable fallback to try if utf-8 failed.